<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
	<class>Ind</class>
	<widget class="QMainWindow" name="Ind">
		<property name="enabled">
			<bool>true</bool>
		</property>
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>351</width>
				<height>410</height>
			</rect>
		</property>
		<property name="minimumSize">
			<size>
				<width>351</width>
				<height>410</height>
			</size>
		</property>
		<property name="maximumSize">
			<size>
				<width>351</width>
				<height>410</height>
			</size>
		</property>
		<property name="windowTitle">
			<string>IG Delete Current Follow Requests</string>
		</property>
		<property name="windowIcon">
			<iconset>
				<normaloff>users.png</normaloff>users.png
			
			</iconset>
		</property>
		<property name="tabShape">
			<enum>QTabWidget::Rounded</enum>
		</property>
		<widget class="QWidget" name="centralwidget">
			<widget class="QLineEdit" name="ig_pwd">
				<property name="geometry">
					<rect>
						<x>182</x>
						<y>30</y>
						<width>161</width>
						<height>32</height>
					</rect>
				</property>
				<property name="placeholderText">
					<string>Password</string>
				</property>
			</widget>
			<widget class="QLineEdit" name="ig_user">
				<property name="geometry">
					<rect>
						<x>10</x>
						<y>30</y>
						<width>161</width>
						<height>32</height>
					</rect>
				</property>
				<property name="placeholderText">
					<string>Username</string>
				</property>
			</widget>
			<widget class="QPushButton" name="login_and_fetch">
				<property name="geometry">
					<rect>
						<x>10</x>
						<y>70</y>
						<width>331</width>
						<height>34</height>
					</rect>
				</property>
				<property name="text">
					<string>Login and Fetch Current Follow Requests</string>
				</property>
			</widget>
			<widget class="QPushButton" name="delete_selected">
				<property name="geometry">
					<rect>
						<x>10</x>
						<y>330</y>
						<width>111</width>
						<height>34</height>
					</rect>
				</property>
				<property name="text">
					<string>Delete Selected</string>
				</property>
			</widget>
			<widget class="QPushButton" name="delete_all_cfr">
				<property name="geometry">
					<rect>
						<x>230</x>
						<y>330</y>
						<width>111</width>
						<height>34</height>
					</rect>
				</property>
				<property name="text">
					<string>Delete All</string>
				</property>
			</widget>
			<widget class="QProgressBar" name="cfr_progress">
				<property name="geometry">
					<rect>
						<x>0</x>
						<y>0</y>
						<width>351</width>
						<height>23</height>
					</rect>
				</property>
				<property name="value">
					<number>0</number>
				</property>
				<property name="alignment">
					<set>Qt::AlignCenter</set>
				</property>
				<property name="textDirection">
					<enum>QProgressBar::TopToBottom</enum>
				</property>
			</widget>
			<widget class="QListWidget" name="cfrs">
				<property name="geometry">
					<rect>
						<x>10</x>
						<y>110</y>
						<width>331</width>
						<height>211</height>
					</rect>
				</property>
			</widget>
			<widget class="QPushButton" name="log_out_exit">
				<property name="geometry">
					<rect>
						<x>10</x>
						<y>370</y>
						<width>331</width>
						<height>34</height>
					</rect>
				</property>
				<property name="text">
					<string>Log Out and Exit</string>
				</property>
			</widget>
			<widget class="QPushButton" name="save_cfrs">
				<property name="geometry">
					<rect>
						<x>130</x>
						<y>330</y>
						<width>91</width>
						<height>34</height>
					</rect>
				</property>
				<property name="text">
					<string>Save List</string>
				</property>
			</widget>
		</widget>
	</widget>
	<resources/>
	<connections/>
</ui>
